transport_analysis/viscosity.py

```diff
@@ -203,7 +209,7 @@ def _conclude(self):
             )
 
             # square and sum each x(, y, z) diff per particle
-            sq_diff = np.square(diff).sum(axis=-1)
+            sq_diff = np.square(diff).mean(axis=-1)
```

Comment:
@hmacdope @xhgchen, this is the most significant change in this PR, it fixes a bug in the initial implementation.

Previously, we were calculating viscosity in the X, Y, and Z directions and then summing the results, I believe the correct behavior is to take the mean of all three. 

Viscosity should be invariant to rotation, thus viscosity in X and Y and Z should all be the same. It does not make sense in my mind to take the sum of all three. Additionally, the values are much closer to the benchmarking data when we take the mean.

Reply:
Yep thats an implementation error. Thanks for picking this up.

Reply:
Thanks for the sanity check.

Reply:
Great catch! Thanks for the fix and improvements @orionarcher! Appreciate the review, @hmacdope!